lib/helper_patches.rb

```diff
@@ -0,0 +1,15 @@
+# -*- encoding : utf-8 -*-
+
+module ApplicationHelper
+
+  def province_name_options
+    province_names = if FastGettext.locale == "nl_BE"
+      %w(Antwerpen Brussel Limburg Oost-Vlaanderen
+         West-Vlaanderen Vlaams\ Brabant Andere)
+    else
+      %w(Brabant\ Wallon Bruxelles Hainaut Liège Luxembourg Namur Autre)
```

Comment:
I think these should belong to `User` so that we can check that a valid province has been entered. I think we should `allow_blank: true` so that this and postcode are optional, and also strip attributes on both so that we don't get empty strings saved in the DB